firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-# Unreleased
+# Unreleased (21.1.0)
+- [changed] With disk persistence enabled, query processing time for most
+  queries is now mainly dependent on the size of the result set rather than the
```

Comment:
I was worried about over-promising and now you're making an even stronger promise. 😄   I think this needs some caveat like "repeatedly-executed queries" or something, since the first time you do a query it will still be slow.

Reply:
I tried walk back on the promise in the second sentence. To make it clearer, I changed it back to also use "repeatedly executed".